README.md

```diff
@@ -0,0 +1,701 @@
+
+If a `receiver` is defined by a group id then an `authorizer` must be able to resolve the group to a list of public keys containing only the current group members. The public key of any `invoker` must be a member of this group.
+
+If the `receiver` contains wildcard symbol `*` then this capability can be invoked by _any_ peer.
+
+## `subject`
+
+Peer who is the owner of the resource this capability targets and therefore the root of any delegation chain.
+
+## `action`
+
+Action a capability gives authority to perform. Can be any string value using `/` as separator between arguments, initially supported values are:
+
+### Read actions
+
+- `document/read`: read and sync a document
+
+### Write actions
+
+- `document/write`: write to a document
+- `document/delete`: delete a document
+- `collection/add`: add documents to a collection
+- `collection/remove`: remove documents from a collection
+- `group/add`: add members to a group
+- `group/remove`: remove members from a group
+- `capability/issue`: issue capabilities
+- `capability/revoke`: revoke capabilities
+
+Actions can be application specific with desired level of granularity. For example, if a document contains a Yjs CRDT Map, you may want to authorize peers to only be able to write to one key. This could be achieved by introducing the action `ycrdt/write/<KEY>`, where `KEY` is the only field allowed to write to. This could be coupled with custom document methods and validation logic in order to implement capabilities providing a granularity of access control which is finer than the document boundary itself.
+
+## `conditions`
+
+Conditions are how the authority a capability delegates is bounded, adding conditions can restrict a capability along several parameters. Supported conditions are:
+
+| Field            | Type         | Action     | Required | Description                                                       |
+| ---------------- | ------------ | ---------- | -------- | ----------------------------------------------------------------- |
+| `document_ids`   | `[Hash]`     | read/write | No       | List document ids this capability applies to                      |
+| `schema_ids`     | `[SchemaId]` | read/write | No       | List of schema ids this capability applies to                     |
+| `to_timestamp`   | `Integer`    | read/write | No       | The timestamp until which read/write of operations is authorized  |
+| `from_timestamp` | `Integer`    | read/write | No       | The timestamp from which read/write of operations is authorized   |
+| `to_seq`         | `Integer`    | write      | No       | The seq number until which read/write of operations is authorized |
+| `from_seq`       | `Integer`    | write      | No       | The seq number from which read/write of operations is authorized  |
+
+### Empty conditions
+
+If the conditions are empty then the action contained in the capability is authorized for any document owned by the `subject` &/ `issuer`.
```

Comment:
Confusing to me. The subject is a document typically, not an owner (peer/group)?

is &/ meant to symbolize something?

Reply:
The subject is a peer or group, this is different from the UCAN spec where a subject can be the resource itself. Hmmm, that should be improved maybe (the naming) so that it is clearly not the same.